src/CoreWCF.RabbitMQ/tests/IntegrationTests.cs

```diff
@@ -28,13 +28,31 @@ public IntegrationTests(ITestOutputHelper output)
         }
 
         [Fact(Skip = "Requires RabbitMQ host with SSL")]
-        public void ClassicQueueWithTls_ReceiveMessage_Success()
+        public void ClassicQueueWithTls_SendReceiveMessage_Success()
         {
```

Comment:
You should probably add the Trait for LinuxOnly on here too. Is the reason for lack of SSL because the container doesn't come with SSL configured? If so, can you open an issue to enable this scenario?

Reply:
This got me thinking, is there a mechanism to provide a certificate validation callback to the RabbitMQ client library? We can easily generate a certificate for the server, but if we don't install the CA cert on the test machine, the certificate will be rejected. If there is a certificate validation callback mechanism, we should get the certificate validator from the ClientCredentials and hook it up.

Reply:
1. Added the LinuxOnly trait
2. Yes, my understanding is that we will need to provide a cert and mount the volume containing the cert ([example](https://gitlab.com/hito20/rabbitmq-with-tls-in-docker/-/blob/393d06135c6ed09b6d457b207fe34b9de0e157e2/docker-compose.yml)). 
3. Yes, the RabbitMQ client does have the validation callback mechanism defined [here](https://rabbitmq.github.io/rabbitmq-dotnet-client/api/RabbitMQ.Client.SslOption.html#RabbitMQ_Client_SslOption_CertificateSelectionCallback).

Reply:
Issue created here: https://github.com/CoreWCF/CoreWCF/issues/1047

Reply:
The remote certificate validation configuration from the ClientCredentials needs to be wired up. The code for [getting the validator is here](BufferManagerOutputStream). The code to convert this to a delegate which matches RemoteCertificateValidationCallback can be found [here](AddServerCertIdentityValidation). You can just copy that method removing some of the tracing code which you don't need.

Reply:
The 2 pull request links are returning 404 errors for me. Is there PR # associated with them that I could try using instead?

Reply:
I don't know what happened but I think it's fixed now

Reply:
Links work now, thanks!

Reply:
Is there an alternate method to [get the cert validator](https://github.com/dotnet/wcf/blob/main/src/System.ServiceModel.Http/src/System/ServiceModel/Channels/HttpsChannelFactory.cs#L32)? It appears to be [internal](https://github.com/dotnet/wcf/blob/main/src/System.ServiceModel.Primitives/src/System/ServiceModel/Security/X509ServiceCertificateAuthentication.cs#L151).